README.adoc

```diff
@@ -357,15 +357,14 @@ Run the `mvn package` command and refresh the {openapi-url} URL to see the updat
 
 == Testing the service
 
-A few tests are included for you to test the basic functionality of the `inventory` service, but they
-do not test the correctness of the generated OpenAPI document. If a test failure occurs, then you might
-have introduced a bug into the code. These tests will run automatically as a part of the Maven build
-process when you run the `mvn install` command. You can also run these tests separately from the build
-by using the `mvn verify` command, but first make sure that the server is stopped.
-
 No explicit tests exist to verify the correctness of the generated OpenAPI document. Manually verify
```

Comment:
No explicit tests exist is too strong a statement.  It seems to imply that no test exists in the world for this.  I would consider something like no automated tests are provided.

Reply:
ok, fixed